### ISSUE TITLE
Make losses same as metrics?

### ISSUE THREAD

**Issue by maxcw** [idealo/image-super-resolution#226]
In the training object, the default losses are MSE, binary cross-entropy, and MAE. Is this what the models are minimizing? If so, how can we make the losses the same as the super-resolution metrics? Why aren't they the super-resolution metrics like PSNR and perceptual loss?
